package.json

```diff
@@ -91,7 +91,7 @@
     "apollo-client": "^1.4.2",
     "archiver": "^1.1.0",
     "async": "^1.5.2",
-    "aws-sdk": "^2.7.13",
```

Comment:
Could you update the `package-lock.json` file as well?

Reply:
Certainly. Is there a proper way to do it? I haven't yet figured out exactly how npm manages that stuff.

Reply:
Trust me. I faced this situation a couple of times and I don't know the correct answer to this question (if there's even a correct answer...)

Here's what I do all the time: `rm -rf node_modules package-lock.json && npm install` 😅.

Reply:
Done.